README.md

```diff
@@ -1,22 +1,55 @@
-# Vue Storefront PayPal Payment Extension
+# Vue Storefront PayPal Payment Extension using Magento 2
 
-PayPal Payment module for [vue-storefront](https://github.com/DivanteLtd/vue-storefront), by [Develo Design](https://www.develodesign.co.uk).
+PayPal Payment Magento 2 module for [vue-storefront](https://github.com/DivanteLtd/vue-storefront), forked from [Develo Design](https://github.com/develodesign/vsf-payment-paypal).
 
 ![Demo](docs/demo.png)
 
+Tested with 1.10.x
 
-## Installation:
+This module is for those who want to use Magento 2's built in Paypal extension that uses the deprecated NVP/Soap api. Tested on Magento 2.2.x. This allows for backend Magento2 management of the Order lifecycle.
+
+***Note this only tested on US and using region_code, region_id, and region modifications on the cart/order syncTotals functions and in the Checkout Addresses***
+
+This module can also be used with the newer API calls easily, as address and cart items are set to be transferred to Paypal SmartButton.
+
+In Button.vue change:
+
+```js
+mounted () {
+  window.paypal.Buttons({
+    ...
+    createOrder: this.createOrderNvp,
+    ...
+  }).render('.paypal-button')
+},
+```
+
+to
+
+```js
+mounted () {
+  window.paypal.Buttons({
+    ...
+    createOrder: this.createOrderRest,
+    ...
+  }).render('.paypal-button')
+},
+```
+
+## Installation
 
 By hand (preferer):
+
 ```shell
-$ git clone git@github.com:develodesign/vsf-payment-paypal.git ./vue-storefront/src/modules/paypal
+$ git clone git@github.com:tonyisworking/vsf-payment-paypal-magento2.git ./vue-storefront/src/modules/payment-paypal-magento2
 ```
 
 ```json
-"paypal": {
+"paymentPaypalMagento2": {
   "clientId": "",
   "endpoint": {
-    "complete": "http://localhost:8080/api/ext/paypal/complete"
+    "complete": "http://localhost:8080/api/ext/paypal/complete",
```

Comment:
@tonyisworking For the `default.json` need use:
`"complete": "/api/ext/paypal/complete"` without `http://localhost:8080` - this only for  `local.json`